sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BulkItemRequestOptions.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.util.Beta;
+
+/**
+ * Encapsulates options that can be specified for an operation used in Bulk execution. It can be passed while
+ * creating bulk request using {@link BulkOperations}.
+ */
+@Beta(Beta.SinceVersion.V4_8_0)
```

Comment:
Needs to be 4.9.0 now - 4.8.0 was already released - Non-blocking of course - just make sure to update this before merging.